src/System.Console/src/System/ConsolePal.Unix.cs

```diff
@@ -859,6 +875,16 @@ private TerminalKeyInfo(TermInfo.Database db)
                     AddKey(db, TermInfo.Database.KeySRight, ConsoleKey.RightArrow, true, false, false);
                     AddKey(db, TermInfo.Database.KeyUp, ConsoleKey.UpArrow);
 
+                    AddPrefixKey(db, "kLFT", ConsoleKey.LeftArrow);
```

Comment:
I actually initially had them in constants.  But the constants would basically either be named the same as the string or they'd be named after the ConsoleKey that follows, and in either case the constant name doesn't provide more detail.  The value to a constant then would be if the string is used multiple times, but it's not.  So I decided to leave it like this.

Separately, I have on my TODO list to revise how a bunch of this stuff is stored.  It's grown organically as we've added this or that feature, and both the code organization and the run time impact of it have become a bit unwieldy.  I'll be doing that in a separate PR.